src/main/java/org/kcctl/command/PatchOffsetsCommand.java

```diff
@@ -0,0 +1,170 @@
+/*
+ *  Copyright 2021 The original authors
```

Comment:
Dang, can't believe it has been three years already since the inception of kcctl :) Note there's https://github.com/kcctl/kcctl/issues/340 for getting rid of the year in headers.